guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -10,13 +10,21 @@ This information allows Katello to display manifest labels, annotations, and inf
 As a result, migrations must be performed to pull this content from manifests into the database.
 
 ifdef::katello[]
-This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following command on {ProjectServer} for a pre-migration:
+.Procedure
+. This migration takes time, so if you depend on container content and need minimal upgrade downtime, enter the following commands in a `tmux` window on {ProjectServer} for a pre-migration:
```

Comment:
This line actually sort of introduces both steps, right? In that case, it should not be part of either step. It would be a good fit for an introductory paragraph -- either before or `.Procedure`. What do you think?

Reply:
I agree. I am going to move it before the procedure steps.

Reply:
Moved it before the procedure and reworded.